calculator.py

```diff
@@ -0,0 +1,50 @@
+import calendar
+import sys
+import datastructures
+
+
+class Calculator:
+    def __init__(self):
+        self.weather_data: list[datastructures.WeatherData] = []
+
+    def find_highest_in_year(self):
+        max_temperature = float('-inf')
+        day = ""
+        month = ""
+        for data in self.weather_data:
+            if (data.day_data[0].split('-')[0] == sys.argv[2]) \
```

Comment:
user python striptime to make datetime object

and them get day or month from datetime object